src/main.scss

```diff
@@ -0,0 +1,9 @@
+@import "./styles/utils/reset.scss";
```

Comment:
File `main.scss` needs to be placed in `styles` folder

Reply:
fixed